Assets/MixedRealityToolkit/_Core/Inspectors/MixedRealitySpeechCommandsProfileInspector.cs

```diff
@@ -25,12 +25,7 @@ public class MixedRealitySpeechCommandsProfileInspector : MixedRealityBaseConfig
 
         private void OnEnable()
         {
-            if (MixedRealityManager.Instance.ActiveProfile == null)
-            {
-                Debug.LogError("No Active Profile set on the Mixed Reality Manager.");
```

Comment:
Why remove this error?  The MRManager cannot operate without a config, unless you have now replaced this with the popup you mentioned.
Even with the popup however, it is still a valid error if you try to run the project without one

Reply:
Because I added logic to prevent users from going into play mode if no active profile is set.